platform/iphone/main.m

```diff
@@ -42,7 +42,7 @@ int main(int argc, char *argv[]) {
 	gargv = argv;
 
 	NSAutoreleasePool *pool = [NSAutoreleasePool new];
-	AppDelegate *app = [AppDelegate alloc];
+	AppDelegate *app = [[AppDelegate alloc] init];
```

Comment:
Not sure how this worked before.

Reply:
This used to raise a warning, will this fix it?
```
platform/iphone/main.m:45:15: warning: unused variable 'app' [-Wunused-variable,34]
        AppDelegate *app = [AppDelegate alloc];
                      ^
2 warnings generated.
```